index.js

```diff
@@ -202,7 +205,7 @@ module.exports = (options = {}) => (tree) => {
         );
       }, '');
 
-    const code = h('code', {}, u('raw', raw));
+    const code = h('code', { className: `language-${lang}` }, u('raw', raw));
```

Comment:
what if lang is undefined? wouldn't that add `language-undefined`?

Reply:
You are right. Let me fix this.